packages/cardano-services/src/Utxo/DbSyncUtxoProvider/types.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * There will be as much rows as tokens are in Value object
+ */
+export interface UtxoModel {
+  address: string;
+  coin: string;
```

Comment:
```suggestion
  coins: string;
```
Sorry @jvieiro, this is slightly wrong still. In future, please refer to the `core` types when applying names: https://github.com/input-output-hk/cardano-js-sdk/blob/550a29bbee59197fa1a856fbc1ea56ba48e41baa/packages/core/src/Cardano/types/Value.ts#L15